blockbase/theme.json

```diff
@@ -59,9 +59,9 @@
 					"name": "Background"
 				},
 				{
-					"slug": "selection",
+					"slug": "tertiary",
 					"color": "#c2c2c2",
```

Comment:
Similarly, Blockbase should be using [the tertiary color from Blank Canvas](https://github.com/Automattic/themes/blob/1aa5058bcff44951877c2db4f461fd4a46433613/blank-canvas/variables.css#L12). I'm not sure where this value came from.

Reply:
Using the tertiary color originally defined in blockbase for text selection makes text selection really hard to see:
![image](https://user-images.githubusercontent.com/146530/133096017-718048b1-ee86-4b2d-aff8-ce3a70c9413d.png)

And when tertiary is the background makes it impossible to see:
![image](https://user-images.githubusercontent.com/146530/133096249-0e64603b-ec7c-45a1-884a-e9737321e1cb.png)

Reply:
changed